common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -1,24 +1,3 @@
-#k_blackrock = {
```

Comment:
I would make this CoA bigger.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122145117-ad554f00-ce65-11eb-98dd-b5ab48fd6975.png)

</details>

Reply:
made it a litte bigger, can't do much more because otherwise it would be too big for the flags 
![image](https://user-images.githubusercontent.com/75175790/122222355-79544b00-ceb2-11eb-8c6e-2eb35954f34d.png)